project-canvas/src/components/DetailView/Components/StoryPointsEstimateMenu.tsx

```diff
@@ -60,63 +60,75 @@ export function StoryPointsEstimateMenu({
   }
 
   return (
```

Comment:
the Loader wont disappear and keep loading when setting the storyPoints 0

Reply:
fixed